server/server.go

```diff
@@ -161,7 +165,7 @@ func (cs *cribbageServer) ginPostCreatePlayer(c *gin.Context) {
 		}
 		return
 	}
-	c.JSON(http.StatusOK, player)
+	c.JSON(http.StatusOK, network.CreatePlayerResponse(cpr))
```

Comment:
This is interesting. I initially used a `network.CreatePlayerResponse` struct literal, passing over the fields from `cpr`, but the linter told me to do this instead. I suspect this will have to change if/when the structs diverge - maybe we should ignore the linter's warning now?

Reply:
it's fine for now. it's one of those little used golang tricks